### ISSUE TITLE
Discovery map disappears until you move the map

### ISSUE THREAD

**Issue by itaher1995** [hydroshare/hydroshare#2452]
When you toggle a filter, the resources will disappear as expected, but when you untoggle the filter, the resources will not return to the screen until you move the map around. However, sometimes it takes a while for the resources to return.

**Comment by Castronova** [comment]
@itaher1995, I'm unable to reproduce this bug.  Below are my steps:

1. go to the map view on the discovery page
2. select a facet 
3. the map points update
4. deselect the facet 
5. the map points update

Can you check to see if this issue has been resolved?

**Comment by itaher1995** [comment]
@Castronova I tried again and followed your exact steps, and I noticed that the map points do end up updating. However, once you click a facet all the point from the previous search disappear like so:

![image](https://user-images.githubusercontent.com/22031618/33083746-3402bb70-ceae-11e7-9376-d67bee39436b.png)

and then if you click on the screen, drag, etc. the map points that were there before the facet was updated appear like

![image](https://user-images.githubusercontent.com/22031618/33083851-6d48bc9a-ceae-11e7-9c03-dacf33accb8c.png)

Both of these images were taken after I unclicked "Christina Bandaragoda". I don't know if this is just an issue on my machine, but I have tried two browsers and the issue persists. Maybe try dragging or clicking the map when you deselect the facet and see if something changes on the screen.
